server/src/meguca/assets/common.go

```diff
@@ -45,3 +66,83 @@ func (s *FileStore) Get(board string) (file File) {
 	}
 	return file
 }
+
+func generateVideoNames() {
+	rw.Lock()
+	defer rw.Unlock()
+
+	ret := []string{"none"}
+	files, err := ioutil.ReadDir("www/videos")
+
```

Comment:
You have some odd fetishes.

Reply:
What?

Reply:
Oh, good point, no need because it's already locked by a mutex, I can just use the `videoNames` itself.